database.php

```diff
@@ -0,0 +1,7 @@
+<?php
+$connection = mysqli_connect("localhost", "root", "", "books_db");
```

Comment:
**Observations:** The database connection is established using procedural code within a single script.

**Recommendations:**
- **Refactor to Use a Database Connection Class:**
  - Create a `DatabaseConnection` class to handle connection logic and manage the connection lifecycle.
  - Use the Singleton pattern to ensure only one connection instance is used throughout the application.
  - Example:
    ```php
    class DatabaseConnection {
        private static $instance = null;
        private $connection;

        private function __construct() {
            $this->connection = new mysqli("localhost", "root", "", "books_db");
            if ($this->connection->connect_error) {
                die("Failed to connect to database: " . $this->connection->connect_error);
            }
        }

        public static function getInstance() {
            if (self::$instance == null) {
                self::$instance = new DatabaseConnection();
            }
            return self::$instance->connection;
        }
    }
    ```